script.js

```diff
@@ -1,4 +1,336 @@
+var gameMode = "NA";
+var playerTurn = "NA";
+var sequenceToggle = "OFF";
+var player1 = 0;
+var player2 = 0;
+var player1Storage = 0;
+var player2Storage = 0;
+var player1Name = "Player1Name";
+var player2Name = "Player2Name";
+var dice1 = 0;
+var dice2 = 0;
+var dice3 = 0;
+var dice4 = 0;
+
+// Entry of Player's Name
+var enterPlayerName = function (input) {
+  playerName = input;
+  return playerName;
+};
+
+//Dice number generator
+var randomDiceRoll = function () {
+  var randomDiceRoll = Math.ceil(Math.random() * 6);
+  return randomDiceRoll;
+};
+
+//select sequence
+var selectSequence1 = function (dice1, dice2) {
+  var selectSequence1 = dice1 + "" + dice2;
+  return Number(selectSequence1);
+};
+var selectSequence2 = function (dice2, dice1) {
+  var selectSequence2 = dice2 + "" + dice1;
+  return Number(selectSequence2);
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  // Enter Player 1 Name
+  if (player1Name == "Player1Name") {
+    var playerName = enterPlayerName(input);
+    player1Name = playerName;
+    return player1Name + " has joined the game";
+  }
+  // Enter Player 2 Name
+  if (player2Name == "Player2Name") {
+    var playerName = enterPlayerName(input);
+    player2Name = playerName;
+    return player2Name + " has joined the game";
+  }
```

Comment:
smart way of getting players name but it would be better if there is an put invalidation